core-bundle/contao/themes/flexible/styles/basic.css

```
@@ -957,6 +953,11 @@ a.tl_submit {
 		min-height: 80px;
 	}
 
+	.nogrid *:not(.widget-group) > .w25, .nogrid *:not(.widget-group) > .w33, .nogrid *:not(.widget-group) > .w50,
+	.nogrid *:not(.widget-group) > .w66, .nogrid *:not(.widget-group) > .w75 {
+		float: none;
+	}
+
```

```suggestion
	.nogrid :not(.widget-group) > :is(.w25, .w33, .w50, .w66, .w75) {
		float: none;
	}

```

This would work as well and save some CSS but w/e both ways should be fine.
The wildcard `*` isn't needed.
More on the `:is`-selector that is supported in all major browsers now:
https://caniuse.com/css-matches-pseudo
https://developer.mozilla.org/en-US/docs/Web/CSS/:is